en/docs/deploy/security/enable-hsts.md

```diff
@@ -1,18 +1,17 @@
 # Enable HTTP Strict Transport Security (HSTS) Headers
 
-Enable HTTP Strict Transport Security headers (HSTS) for the applications deployed in your server, to confirm that the relevant headers are present in the HTTP response. HSTS is not enabled for applications in WSO2 Identity Server by default. 
+Enable HTTP Strict Transport Security (HSTS) headers for the applications deployed in your server, to confirm that the relevant headers are present in the HTTP response. HSTS is not enabled for applications in WSO2 Identity Server by default. 
 
 !!! note 
-
 	HSTS should not be enabled in development environments because transport security validations can interrupt the development processes by validating signatures of self-signed certificates. 
 
 ---
 
 ## Enable for Management Console 
 
-If the `HttpHeaderSecurityFilter` element is available in the `web.xml` file (stored in the `<IS_HOME>/repository/conf/tomcat/carbon/WEB-INF/` directory) as shown below, it implies that security headers are by default configured for the management consoles of all of your profiles. However, in a production deployment, Strict-Transport-Security needs to be explicitly enabled by replacing the default <init-param> values of the `HttpHeaderSecurityFilter` filter.
+If the `HttpHeaderSecurityFilter` element is available in the `<IS_HOME>/repository/conf/tomcat/carbon/WEB-INF/web.xml` file as shown below, it implies that security headers are by default configured for the WSO2 Management Consoles of all of your profiles. However, in a production deployment, Strict-Transport-Security needs to be explicitly enabled by replacing the default <init-param> values of the `HttpHeaderSecurityFilter` filter.
```

Comment:
`<init-param>` -> is this shown as expected in the UI?

Reply:
Yes